pyvenv.cfg

```diff
@@ -0,0 +1,8 @@
+home = /usr
```

Comment:
This file is not part of the standards documents of an Odoo module.

Reply:
Noted. I'll work on it shortly.